### PR TITLE
landing page update

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 1]
```diff
@@ -193,7 +194,7 @@ export class LandingPage extends ViewPlugin {
           href="https://twitter.com/EthereumRemix"
         >
         </a>
-        <script src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+        <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (yann300) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:197]: why do you need to load the same script twice?

Comment (LianaHus) [reply apps/remix-ide/src/app/ui/landing-page/landing-page.js:197]: because I couln't change data-theme

Comment (LianaHus) [reply apps/remix-ide/src/app/ui/landing-page/landing-page.js:197]: ah you are right

Comment (LianaHus) [reply apps/remix-ide/src/app/ui/landing-page/landing-page.js:197]: or not. I use update of yo-yo

**Review by yann300 — APPROVED** [review]

**Pull request merged**